.scalafmt.conf

```diff
@@ -0,0 +1,10 @@
+version = 2.0.1
+align = none
+align.openParenCallSite = true
+align.openParenDefnSite = true
+maxColumn = 120
```

Comment:
I see lines spanning at least 146 chars here and there for some reason but most are under 120, which is a dream come true for me  😇 .